src/libYARP_dev/src/devices/AnalogSensorClient/AnalogSensorClient.cpp

```diff
@@ -10,13 +10,22 @@
 #include "AnalogSensorClient.h"
 #include <yarp/os/Log.h>
 #include <yarp/os/LogStream.h>
+#include <yarp/os/Value.h>
 
 /*! \file AnalogSensorClient.cpp implementation of an analog sensor client class*/
 
 using namespace yarp::dev;
 using namespace yarp::os;
 using namespace yarp::sig;
 
+namespace
+{
+    inline int checkResponse(bool ok, yarp::os::Bottle& response)
```

Comment:
The signature should be: `inline int checkResponse(bool ok, const yarp::os::Bottle& response)`

Reply:
Right, I'll change this. I copy-pasted this signature from 

https://github.com/robotology/yarp/blob/a211f01990c5de5845f37cc12aea2e09e53febd9/src/libYARP_dev/include/yarp/dev/ControlBoardHelpers.h#L26-L37

Reply:
Done at https://github.com/robotology/yarp/pull/2003/commits/a4909c55c1aedf11575f53fb2bb54f1c41ae7878.